tests/mock_tables/counters_db.json

```diff
@@ -3555,7 +3555,13 @@
     "Ethernet8:12": "oid:0x15000000010254",
     "Ethernet8:13": "oid:0x15000000010255",
     "Ethernet8:14": "oid:0x15000000010256",
-    "Ethernet8:15": "oid:0x15000000010257"
+    "Ethernet8:15": "oid:0x15000000010257",
+    "Ethernet16:0":  "oid:0x15000000000260",
+    "Ethernet16:1":  "oid:0x15000000000261",
+    "Ethernet16:2":  "oid:0x15000000000262",
+    "Ethernet16:3":  "oid:0x15000000000263",
+    "Ethernet16:4":  "oid:0x15000000000264",
+    "Ethernet16:6":  "oid:0x15000000000266"
   },
```

Comment:
Could you also add test data with below situation?
1. an extra port with incomplete MC
2. an extra port with incomplete UC and incomplete MC.